src/pages/MainPage/ContactsSection.jsx

```diff
@@ -23,42 +25,42 @@ const ContactsSection = () => {
             <div className="mt-11 h-32 w-full rounded-10px text-center shadow-[6px_6px_6px_rgba(0,0,0,0.25)]">
               <MailIcon className="mt-5 inline-block" />
               <p className="mt-3 font-fourth text-base">EMAIL</p>
-              <a
-                href="#"
+              <Link
```

Comment:
Why do we need email as a link?

Reply:
I wanted to implement something like mailto:, to make it clickable

Reply:
so in this case `Link` component isn't applicable because it serves for internal routing but `mailto:` is about external route